skyportal/handlers/api/config_handler.py

```diff
@@ -96,8 +98,7 @@ def get(self):
                 "summary_sourcesClasses": cfg["colors.summary_sources"],
                 "tnsAllowedInstruments": TNS_INSTRUMENTS,
                 "gcnTagsClasses": cfg["colors.gcnTags"],
-                "colorPalette": getattr(
-                    palettes, cfg.get("misc.bokeh_color_palette", "Turbo256")
-                ),
+                "colorPalette": COLOR_PALETTE,
```

Comment:
Sounds like a good idea. Originally I felt like not adding more to the config would help with its readability, but it's a nice option to have it be configurable.

Reply:
it's a very long list however. If I do add it to the yaml config, maybe it should at least be on just one line to not double its current length. I'm not sure this palette is meant to be modified. It's grabbing the nice bokeh color palette we were using to avoid keeping bokeh as a dependency just for that.